L/libcleri/build_tarballs.jl

```diff
@@ -0,0 +1,51 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libcleri"
+version = v"0.12.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/transceptor-technology/libcleri.git", "a8a1adc9dcf4e889a4d44c17acb20d74d0c6cfe5")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+export CFLAGS=$(pcre2-config --cflags)
+export LDFLAGS=$(pcre2-config --libs8)
+cd $WORKSPACE/srcdir
+cd libcleri/
+make -C Release -f makefile
+make -C Release -f makefile install INSTALL_PATH=${prefix}
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("i686", "linux"; libc = "glibc"),
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("aarch64", "linux"; libc = "glibc"),
+    Platform("powerpc64le", "linux"; libc = "glibc"),
+    Platform("i686", "linux"; libc = "musl"),
+    Platform("x86_64", "linux"; libc = "musl"),
+    Platform("aarch64", "linux"; libc = "musl"),
+    Platform("x86_64", "macos"; ),
+    Platform("x86_64", "freebsd"; ),
+    Platform("i686", "windows"; ),
+    Platform("x86_64", "windows"; )
+]
+
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libcleri", :libcleri)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="PCRE2_jll", uuid="efcefdf7-47ab-520b-bdef-62a2eaa19f15"))
+]
+
+# Build the tarballs, and possibly a `build.jl` as well.
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6")
```

Comment:
> The error on armv71 is an internal compiler error, details pasted below.

Ugh, that's bad

> For now I simply skipped these platforms, but I'm happy to revisit, for example with newer compiler versions, if that would be preferable.

Yes, I think it should be safe to try here a newer version, for example with
```suggestion
build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version=v"5")
```